### PR TITLE
Add bundle_inject

### PR DESCRIPTION
**Pull request by Fryguy**
@bdunne Please review.

### CODE REVIEW

**Comment by Fryguy** [comment]
I can't understand this one.

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
+plugin 'bundler-inject'
```
Comment (kbrock) [inline Gemfile:3]: did you want to version this? I think most of our spots we have "~> 2.0"

**Review by kbrock — APPROVED** [review]
just that question - otherwise, LGTM

**Comment by Fryguy** [comment]
@kbrock It doesn't make its way to Gemfile.lock, so technically it doesn't matter.  I still can't understand why this fails though.

**Comment by kbrock** [comment]
> @kbrock It doesn't make its way to Gemfile.lock, so technically it doesn't matter. I still can't understand why this fails though.

@Fryguy Is it possible that one of the dependencies mentions this at a different version?

**Comment by kbrock** [comment]
kicking

**Pull request merged**